### PR TITLE
Send path when resolving actions

### PR DESCRIPTION
**Pull request by thboop**
This PR sends up the path in the resolve actions endpoint so we can properly validate policy when JIT downloading Composite Actions

### CODE REVIEW

File: src/Runner.Worker/ActionManager.cs [hunk 1]
```diff
@@ -610,6 +610,7 @@ private async Task BuildActionContainerAsync(IExecutionContext executionContext,
                     {
                         NameWithOwner = repositoryReference.Name,
                         Ref = repositoryReference.Ref,
+                        Path = repositoryReference.Path,
```
Comment (ericsciple) [inline src/Runner.Worker/ActionManager.cs:613]: nit: consider deleting the comment on this method (line 595)

Comment (ericsciple) [reply src/Runner.Worker/ActionManager.cs:613]: `my-org/my-repo/my-folder/my-subfolder@v1`

- NameWithOwner: my-org/my-repo
- Ref: v1
- Path my-folder/my-subfolder

**Review by ericsciple — APPROVED** [review]

**Comment by KyleJamesWalker** [comment]
@thboop or @ericsciple do you think this would be causing our composite actions to be erroring out now?

It was working yesterday just fine, and now we're seeing:
```
Error: `uses:` keyword is not currently supported.
```

I noticed this was part of the release that came out 4 hours ago, we are using a composite action with uses clauses.

**Comment by thboop** [comment]
@KyleJamesWalker were you using custom self hosted runnners by any chance? Composite should never have been enabled for public use on github.com. That being said, we plan to ship the feature in the very near future.

**Comment by KyleJamesWalker** [comment]
@thboop We're using public runners, not sure if it matters any but we are also on GitHub enterprise.

**Comment by KyleJamesWalker** [comment]
In case anyone looks at this later it is now fully working as expected: https://github.com/actions/runner/issues/646#issuecomment-901336347

**Pull request merged**